.bazelrc

```diff
@@ -1,3 +1,20 @@
+#
+# Copyright (C) 2020 Grakn Labs
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
 build --incompatible_strict_action_env
 run --incompatible_strict_action_env
 test --incompatible_strict_action_env
```

Comment:
The bazelrc doesn't have the buildbuddy stuff here. is it expected?

Compare it with the bazelrc of grakn-core for example: https://github.com/graknlabs/grakn-2.0/blob/master/.bazelrc

Reply:
What's missing? It looks good to me

Reply:
ah lol yeah it's actually there when i expanded it. was just delusional